source/portfolio/data_feed/data_feed.h

```diff
@@ -11,18 +11,21 @@
 #include <string_view>
 
 namespace portfolio {
+    using day_point =
+        std::chrono::time_point<std::chrono::system_clock, date::days>;
+    using minute_point = std::chrono::time_point<std::chrono::system_clock,
+                                                 std::chrono::minutes>;
+    using interval_points = std::pair<minute_point, minute_point>;
+    using ohlc_prices = std::tuple<double, double, double, double>;
```

Comment:
Se for reutilizar muito esta estrutura, que provavelmente vai, pode criar um novo header com o template para ele. Isto pode ir para `<portfolio/core/ohlc_prices.h>`.

Assim, você pode passar apenas um tipo para o template e acessar os dados .open(), .high(), etc... em vez de std::get<0>, std::get<1>, ...

Isso também te permite adicionar novas operações que descobrir para ohlcs com o tempo.

Reply:
Criei a classe ohlc_prices conforme sugestão.